core/drivers/crypto/hisilicon/sec_main.c

```diff
@@ -0,0 +1,266 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2022-2024 HiSilicon Limited.
+ * Kunpeng hardware accelerator sec module init.
+ */
+#include "sec_main.h"
```

Comment:
IMHO this should also explicitly include _trace.h_, _sys/queue.h_n, _util.h_, _io.h_, _mm/core_memprot.h_, _malloc.h_ and _tee_api_types.h_.

Reply:
Thanks for the advice. These header files are generic header files for SEC and HPRE, I will place these files in hisi_qm.h

Reply:
These should be placed in the source file, not the header file, IMO. I let our dear maintainers give their view.

Reply:
Please listen to @etienne-lms's advice.